fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -368,8 +368,34 @@ def self.hide_loading_indicator
 
     # checks if a given path is an executable file
     def self.executable?(cmd_path)
-      # no executable files on Windows, so existing is enough there
-      cmd_path && !File.directory?(cmd_path) && (File.executable?(cmd_path) || (self.windows? && File.exist?(cmd_path)))
+      if !cmd_path || File.directory?(cmd_path)
+        return false
+      end
+
+      if self.windows?
+        return File.exist?(get_executable_path(cmd_path))
+      else
+        return File.executable?(cmd_path)
+      end
```

Comment:
Can this simplified to 👇? `get_executable_path` looks like it already includes the windows check

```suggestion
      return File.exist?(get_executable_path(cmd_path))
```